### ISSUE TITLE
ClassCastException when using together with Code with me

### ISSUE THREAD

**Issue by szpak** [chashnikov/IntelliJ-presentation-assistant#42]
JetBrains [introduced](https://www.jetbrains.com/help/idea/code-with-me.html) the brand new plugin for remove pair programming. When I share my screen with other peers with IntelliJ-presentation-assistant enabled I observe the following exception:

```
java.lang.ClassCastException: class com.jetbrains.rdserver.ui.popup.bePopups.impl.BackendComponentBePopupBuilder cannot be cast to class com.intellij.ui.popup.ComponentPopupBuilderImpl (com.jetbrains.rdserver.ui.popup.bePopups.impl.BackendComponentBePopupBuilder is in unnamed module of loader com.intellij.ide.plugins.cl.PluginClassLoader @1b47be96; com.intellij.ui.popup.ComponentPopupBuilderImpl is in unnamed module of loader com.intellij.util.lang.UrlClassLoader @1efbd816)
	at org.nik.presentationAssistant.ActionInfoPanel.<init>(ActionInfoPanel.kt:63)
	at org.nik.presentationAssistant.ShortcutPresenter.showActionInfo(ShortcutPresenter.kt:141)
	at org.nik.presentationAssistant.ShortcutPresenter$enable$1.beforeActionPerformed(ShortcutPresenter.kt:66)
	at jdk.internal.reflect.GeneratedMethodAccessor166.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.intellij.util.messages.impl.MessageBusImpl.invokeListener(MessageBusImpl.java:632)
	at com.intellij.util.messages.impl.MessageBusImpl.executeOrAddToQueue(MessageBusImpl.java:245)
	at com.intellij.util.messages.impl.CompositeMessageBus$ToDirectChildrenMessagePublisher.publish(CompositeMessageBus.java:115)
	at com.intellij.util.messages.impl.MessageBusImpl$MessagePublisher.invoke(MessageBusImpl.java:183)
	at com.sun.proxy.$Proxy187.beforeActionPerformed(Unknown Source)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.fireBeforeActionPerformed(ActionManagerImpl.java:1479)
	at com.jetbrains.rdserver.actions.BackendActionExecutorDelegate$tryInvokeAction$$inlined$fullSuppressMarkupSynchronization$lambda$1$1.invoke(BackendActionExecutorDelegate.kt:75)
	at com.jetbrains.rdserver.actions.BackendActionExecutorDelegate$tryInvokeAction$$inlined$fullSuppressMarkupSynchronization$lambda$1$1.invoke(BackendActionExecutorDelegate.kt:24)
	at com.jetbrains.rdserver.ui.feelLuckUi.BackendFeelLuckyUiHelper.executeWithUiInteractionMarker(BackendFeelLuckyUiHelper.kt:17)
	at com.jetbrains.rdserver.actions.BackendActionExecutorDelegate$tryInvokeAction$$inlined$fullSuppressMarkupSynchronization$lambda$1.invoke(BackendActionExecutorDelegate.kt:74)
	at com.jetbrains.rdserver.actions.BackendActionExecutorDelegate$tryInvokeAction$$inlined$fullSuppressMarkupSynchronization$lambda$1.invoke(BackendActionExecutorDelegate.kt:24)
	at com.jetbrains.rd.ide.requests.IdeGateway.executeBulkUpdate(IdeGateway.kt:65)
	at com.jetbrains.rdserver.actions.BackendActionExecutorDelegate.tryInvokeAction(BackendActionExecutorDelegate.kt:72)
	at com.jetbrains.rdserver.actions.BackendActionExecutorDelegate.execute(BackendActionExecutorDelegate.kt:43)
	at com.jetbrains.rdserver.actions.BackendActionExecutorDelegate.execute(BackendActionExecutorDelegate.kt:24)
	at com.jetbrains.rdserver.requests.BackendAsyncRequestExecutor.executeRequests(BackendAsyncRequestExecutor.kt:176)
	at com.jetbrains.rdserver.requests.BackendAsyncRequestExecutor.access$executeRequests(BackendAsyncRequestExecutor.kt:17)
	at com.jetbrains.rdserver.requests.BackendAsyncRequestExecutor$1.invoke(BackendAsyncRequestExecutor.kt:23)
	at com.jetbrains.rdserver.requests.BackendAsyncRequestExecutor$1.invoke(BackendAsyncRequestExecutor.kt:17)
	at com.jetbrains.rd.util.reactive.Signal.fire(Signal.kt:30)
	at com.jetbrains.rd.framework.impl.RdSignal.onWireReceived(RdSignal.kt:39)
	at com.jetbrains.rd.platform.protocol.SecureWireOverStreamTransport$advise$wrapper$1.onWireReceived(SecureWireOverStreamTransport.kt:75)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:31)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2$2.invoke(MessageBroker.kt:11)
	at com.jetbrains.rd.framework.impl.ProtocolContexts.readMessageContextAndInvoke(ProtocolContexts.kt:151)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:30)
	at com.jetbrains.rd.framework.MessageBroker$invoke$2.invoke(MessageBroker.kt:11)
	at com.jetbrains.rdserver.startup.EdtScheduler$queue$1.run(EdtScheduler.kt:39)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:201)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:802)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:322)
	at com.intellij.codeWithMe.ClientId$Companion.withClientId(ClientId.kt:109)
	at com.intellij.codeWithMe.ClientId.withClientId(ClientId.kt)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:82)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:132)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:47)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:188)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:971)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:841)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:452)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:744)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:451)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:802)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:499)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

Maybe there is some interface which could be used to make it work in both cases.

**Comment by chashnikov** [comment]
Should be fixed by 5d48b398b9eb.

**Comment by chashnikov** [comment]
I've published version 1.0.7 which contains the fix.
